src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "TrackSeeding.h"
 
+#include "TVector2.h"
```

Comment:
Same nitpicky comment here. I guess the advantage here is that you can use the `DeltaPhi` function in `TVector2` for which I don't think eigen has something analogous (a quick scan through [their quick help list](https://eigen.tuxfamily.org/dox/group__QuickRefPage.html) shows only basic trig functions. So if it is possible to use the eigen vector methods here, I would say we should - otherwise can use TVector2

Reply:
This is not nitpicky: We should not be adding a dependency on TVector2, which already is a **legacy** class in ROOT (https://root.cern.ch/doc/master/group__Physics.html). Use Acts::Vector as @osbornjd indicates. Use [ROOT::Math::Cartesian2D](https://root.cern.ch/doc/master/classROOT_1_1Math_1_1Cartesian2D.html) if the context is not Acts-related (does not apply here).